.github/workflows/examples.yml

```diff
@@ -12,6 +12,10 @@ on:
       pytest_args:
         description: 'Pytest arguments'
         default: ''
+      skip_windows:
+        description: 'Skip tests on Windows'
+        type: boolean
+        default: true
```

Comment:
Could you briefly explain how to use this option and why this default value is true?

Reply:
To dont run windows job by default for manual run

Reply:
I would suggest to set false by default.

Reply:
I made it as true, to avoid unnecessary launch on windows:
- it's longer
- linux tests covered 99% falls
- 99% changes is no OS specific

In my mind, windows checks should be run only if:
- add new examples
- changes os specific part

Reply:
Let's discuss it offline.

Reply:
Changed default value for skip_windows to false